modules/alsa/alsa_play.c

```diff
@@ -67,12 +67,12 @@ static void *write_thread(void *arg)
 
 		sampv = st->sampv;
 
-		n = snd_pcm_writei(st->write, sampv, samples);
+		n = (int)snd_pcm_writei(st->write, sampv, samples);
```

Comment:
is it better to change the type of n to `long n` ?

in this case no casting is needed.

Reply:
good suggestion. changed now.